packages/distros/rke2/zarf.yaml

```diff
@@ -0,0 +1,49 @@
+kind: ZarfInitConfig
+metadata:
+  name: "Zarf Init Package Component - RKE2"
```

Comment:
The validator for the package name is pretty strict.. Right now it only allows package names to be all lowercase and contain no special characters other than '-'.

https://regex101.com/r/vpi8a8/1

@jeff-mccoy Could you help me understand why this is so strict? Right now the validator doesn't let numbers in package names either.